kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -434,6 +439,24 @@ protected void processFormLayout(FormInstance topLevelForm,
                                     }
                                     item.setValue((value != null) ? value.toString() : "");
                                     break;
+                                case "util-date":
+                                    if (value != null && value instanceof java.util.Date) {
+                                        if (!field.isShowTime() && value != null && value.toString().length() >= 10) {
```

Comment:
I think here there's no need to check if value is null since it's already checked in #443. I'm not confortable by checking the string length. I think that here we could go just by checking if the `field.isShowTime()` and then do the proper conversion.